tools/accuracy_checker/accuracy_checker/annotation_converters/inpainting.py

```diff
@@ -0,0 +1,85 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..config import PathField, NumberField
+from ..utils import get_path, get_size_from_config
+from ..logging import warning
+from ..representation import ImageInpaintingAnnotation
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class InpaintingConverter(BaseFormatConverter):
+    __provider__ = 'inpainting'
+    annotation_types = (ImageInpaintingAnnotation,)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'images_dir': PathField(
+                optional=False, is_directory=True,
+                description="Path to directory with images."
+            ),
+            'size': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination size for image."
+            ),
+            'dst_height': NumberField(
+                optional=True, value_type=int,
+                description="Destination height for image cropping."
+            ),
+            'dst_width': NumberField(
+                optional=True, value_type=int,
+                description="Destination width for image cropping."
+            ),
+            'masks_dir': PathField(
```

Comment:
on my view, mask should be required for this task.
I do not see the preprocessing which do not use it.